src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java

```diff
@@ -385,7 +385,17 @@ private int loadInternal (Table table) throws Exception {
                 String string = csvReader.get(f);
                 // Use spec table to check that references are valid and IDs are unique.
                 Set<NewGTFSError> errors = table.checkReferencesAndUniqueness(keyValue, lineNumber, field, string, referenceTracker);
-                errorStorage.storeErrors(errors);
+                if (
+                    table.name.equals("calendar_dates") &&
+                    "1".equals(csvReader.get(Field.getFieldIndex(fields, "exception_type")))
+                ){
+                    // Do not record bad reference errors for calendar date entries that add service
+                    // (exception type=1) because a corresponding service_id in calendars.txt is not required in
+                    // this case.
+                    LOG.info("Skipping bad reference for calendar_date adding service (service_id={}).", keyValue);
+                }
+                // In all other cases (outside of the calendar_dates special case), store the reference errors found.
+                else errorStorage.storeErrors(errors);
```

Comment:
Since the above if statement has braces, please include it with this else statement for readability.

Reply:
A convention we have followed for a few years is that braces may be omitted if the statement is on the same line as the conditional keyword. Sometimes that makes things easier to read. But generally I agree, I'd default to wrapping all conditional bodies in braces.